ZuluContent/Items/Weapons/GMWeapons/OmerosPickaxe.cs

```diff
@@ -0,0 +1,81 @@
+using Server.Engines.Harvest;
+using ZuluContent.Zulu.Engines.Magic;
+
+namespace Server.Items
+{
+    public class OmerosPickaxe : BaseAxe, IUsesRemaining, IEnchanted
+    {
+        public override HarvestSystem HarvestSystem
+        {
+            get => Mining.System;
```

Comment:
There's a shorter way if you're only using `get`:

```csharp
        public override HarvestSystem HarvestSystem => Mining.System;
```

Reply:
Ah cool thx for pointing it out